engine/crates/integration-tests/src/postgres.rs

```diff
@@ -208,6 +208,24 @@ impl TestApi {
         .await
     }
 
+    pub async fn execute_parameterized(
+        &self,
+        operation: impl AsRef<str>,
+        variables: impl serde::Serialize,
+    ) -> Response {
+        Box::pin(
+            self.inner
+                .engine
+                // this prevents a race. we initialize the engine only when executing the first request,
+                // so the introspection runs only after we've modified the database schema.
```

Comment:
I don't get this change. Where does introspection come into play in the new tests?